controllers/storagecluster/cephcluster_test.go

```diff
@@ -102,6 +103,50 @@ func TestEnsureCephCluster(t *testing.T) {
 	}
 }
 
+func TestCephClusterMonTimeout(t *testing.T) {
```

Comment:
Since this test is not required to pass the unit tests of the code changes, add the test in a separate commit.

Reply:
Done